### DISCUSSION TITLE
我必须立刻展示刚到货的周边

### DISCUSSION THREAD

**Discussion by qishaming-mihomo** [share121/inter-knot#2305] [其他]
![f61ecbe33e20834c7df60d3c0f025756](https://github.com/user-attachments/assets/7012b676-474d-4a0a-9d75-744f2987d480)
![5d8a29b4bf8a188c9030bbe1b06ed2af](https://github.com/user-attachments/assets/7edd617f-eaf3-4866-8d51-8b4f972ce696)
![290bbe73428061d514eb1995516e2bc1](https://github.com/user-attachments/assets/359fec6d-5bee-4c61-9ed6-bc677e301e9a)
实物真的好看，拍不出来的好看

**Comment by mengsheng521** [comment]
我也要吃谷子

**Comment by Genshin-Neuvillette** [comment]
好看！
